frame/staking/src/benchmarking.rs

```diff
@@ -407,35 +407,55 @@ benchmarks! {
 	}
 
 	submit_solution_initial {
-		let v in 1 .. 10;
-		let n in 1 .. 100;
+		// NOTE: this benchmark requires: `v > w`, `n > a` and `a > w`. Talk to @kianenigma if you
+		// want to know why.
+		// number of validator intentions
+		let v in 50 .. 100;
+		// number of nominators
+		let n in 200 .. 500;
+		// number of winners.
+		let w in 10 .. 40;
+		// number of assignments. Basically, number of active nominators.
+		let a in 50 .. 200;
+		// NOTE: we could make the edge per assignment also variable, but 1. unlikely to bring much
+		// accuracy change 2. makes the setup of the bench a few orders of magnitude more annoying.
```

Comment:
maybe another implementation would just instead of trim a solution just craft a solution:
```
let n in 1..200;
// you create LIMIT (=16) new validators (winners)
// you create n new nominators which nominates only the winners
// you submit a solution with the winnres and all the new nominators with 16 nomination.
```
thus we benchmark for a solution with worst case edge_count and depending on its number of nominator

But I'm ok with this solution

Reply:
We are allowed to actually compute the edge count from the solution (`compact.edge_count()`), should we then still assume worse case?

Reply:
yes edge_count is good here